packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1144,6 +1144,54 @@ export const formatters: Formatters = {
     format: ({ user, desktop_addr, directory_name, file_path, length }) =>
       `User [${user}] failed to write [${length}] bytes to file [${file_path}] in shared directory [${directory_name}] on desktop [${desktop_addr}]`,
   },
+  [eventCodes.DEVICE_CREATE]: {
+    type: 'device',
+    desc: 'Device Register',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has registered a device`
```

Comment:
FYI, in practice the only ones we issue failures for are TV005I (enroll) and TV006I (authn). It's best to keep the tests and the failure strings, in case it changes, but I'm mentioning it in case you want to leave a comment somewhere.